AUTHORS

```diff
@@ -68,6 +68,9 @@ ScummVM Team
     BBVS:
        Benjamin Haisch
 
+    Buried:
+       Matthew Hoops
+
```

Comment:
You should consider to add yourself to the list so that users know they should report issues to you